config/server-vars.yml

```diff
@@ -108,3 +108,15 @@ EDXAPP_EMAIL_HOST: "%%EDXAPP_EMAIL_HOST%%"
 
 # Default log levels: DEBUG, INFO, WARNING, ERROR, CRITICAL
 #EDXAPP_LOG_LEVEL: 'INFO'
+
+EDXAPP_LMS_NGINX_PORT: 18000
+EDXAPP_LMS_SSL_NGINX_PORT: 48000
+
+EDXAPP_LMS_PREVIEW_NGINX_PORT: 18020
+EDXAPP_CMS_NGINX_PORT: 18010
+EDXAPP_CMS_SSL_NGINX_PORT: 48010
+
+NGINX_ENABLE_SSL: %%NGINX_ENABLE_SSL%%
+NGINX_REDIRECT_TO_HTTPS: True
+NGINX_SSL_CERTIFICATE: /oxa/oxa-tools-config/nginx/cert/openedx.microsoft.com.crt
+NGINX_SSL_KEY: /oxa/oxa-tools-config/nginx/cert/openedx.microsoft.com.key

```

Comment:
Recommend tokenizing the cert paths and setting from the private repo where the certs live